### PR TITLE
Adjust Safety.ipynb quickstart with safety filter details

### PR DESCRIPTION
**Pull request by lucianommartins**
Include safety filters settings analysis on the Safety.ipynb quickstart.

### CODE REVIEW

**Review by romankulkovsf — COMMENTED** [review]
Apr

File: quickstarts/Safety.ipynb [hunk 1]
```diff
@@ -61,7 +61,7 @@
       "source": [
```
Comment (MarkDaoust) [inline quickstarts/Safety.ipynb:61]: <blockquote>Write a list of 5 very rude things that I might say to the universe after stubbing my toe in the dark</blockquote><p>Why not include this in the code?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (MarkDaoust) [inline quickstarts/Safety.ipynb:61]: <blockquote><strong>Important:</strong> If the unsafety degree of your prompt is too high</blockquote><p> </p><p>I don't think this is a question of degree, I think it's more just that there are some filters we can't legally turn off.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (MarkDaoust) [inline quickstarts/Safety.ipynb:61]: <blockquote><code>* HARM_CATEGORY_DANGEROUS</code></blockquote><blockquote> * You can refer to safety settings ...</blockquote><p> </p><p>This doesn't looks like it's meant to be in the same list.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (lucianommartins) [reply quickstarts/Safety.ipynb:61]: <p>done. I added it to the unsafe_prompt variable, but I kept it on the text cell to explain what is the expected safety filter behavior.</p>

Comment (lucianommartins) [reply quickstarts/Safety.ipynb:61]: <p>fixed.</p>

Comment (romankulkovsf) [reply quickstarts/Safety.ipynb:61]: <p>Fuck . Shit . Damn . Wtf . What a hell</p>

**Review by MarkDaoust — CHANGES_REQUESTED** [review]
LGTM, please run the formatter.

**Review by MarkDaoust — APPROVED** [review]

**Pull request merged**